### PR TITLE
Prediction code cleanup

### PR DESCRIPTION
**Pull request by simra**
updated the code for local predictions

### CODE REVIEW

File: src/model/predictions/predict.py [hunk 1]
```diff
@@ -1,61 +1,61 @@
...
-                else:
-                    boxesOut.append(box)
-                    canvas.rectangle(box, outline='red', width=3)
-
-            for box, _, label in list(sorted(primers, key=(lambda x: x[1]), reverse=True)):
-                if any(map(lambda x: isContained(box, x), boxesOut)):
-                    if not any(map(lambda x: isRectangleOverlap(box, x), primersOut)):
-                        primersOut.append(box)
-                        canvas.rectangle(box, outline='yellow', width=3)
-                    return dst
```
Comment (monkeypants) [inline src/model/predictions/predict.py:41]: I was wondering where this went... but I see it was duplicated in `inference.Inference.predict()` :)

File: src/model/predictions/predict.py [hunk 2]
```diff
@@ -1,61 +1,61 @@
...
-    with torch.no_grad():
-        img = Image.open(image_path).convert('RGB')
-        img, _ = transform(img, {'image_id':0,  'boxes':[],  'annotations':[]})
-        prediction = model([img.to(device)])
-        masks = [p['masks'] for p in prediction]
-        prediction = [dict([(k, v.cpu().numpy().tolist()) for k, v in x.items() if k != 'masks']) for x in prediction]
-        print(masks, prediction)
-        dst = predict(img, prediction)
-        fn = os.path.join(folder, 'predictions.png')
-        dst.save(fn)
```
Comment (monkeypants) [inline src/model/predictions/predict.py:61]: and so is this (duplicated in `inference.Inference.init()`)

**Review by monkeypants — APPROVED** [review]
I see the clean-up (LGTM) but don't know enough about the ML side to provide a meaningful review of that. I suppose it hasn't changed much so it's probably good to merge :)

File: src/model/predictions/predict.py (hunk 1 above)
Comment (simra) [reply src/model/predictions/predict.py:41]: Yes, all this code was moved to the Inference class- it doesn't make sense to keep a duplicate that might diverge.

**Pull request merged**